sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -234,17 +235,21 @@ interface ReceiveMessageArgs {
  * @ignore
  */
 export class BatchingReceiverLite {
+  private _createAndEndProcessingSpan: typeof createAndEndProcessingSpan;
```

Comment:
Why this? Why not use `createAndEndProcessingSpan` directly?

Reply:
I'll add a comment - it's just to allow us to mock it for unit testing.

Reply:
Expected!